podman/tinyproxy/dev.sh

```diff
@@ -0,0 +1,3 @@
+docker build . -t tp
+docker run -p 23128:8888 --name=tp -v $(pwd)/conf/:/usr/local/etc/tinyproxy/:z tp
```

Comment:
Getting mixed messages if this is hardcoded in the image or a volume...

Reply:
Both!  @ahbarnz-g has some use cases in mind.